dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -6223,6 +6223,14 @@ void CalpontSystemCatalog::buildSysColinfomap()
   aCol.colPosition++;
   aCol.columnOID = OID_SYSCOLUMN_NEXTVALUE;
   fColinfomap[aCol.columnOID] = aCol;
+
+  aCol.colWidth = 4;
```

Comment:
IMHO this is an example how the previously set attributes of a shared class instance can be reused accidentally. Despite this is a pattern I suggest to create another ColType instance inside a scope.